x-pack/test/api_integration/apis/metrics_ui/snapshot.ts

```diff
@@ -37,7 +38,7 @@ export default function ({ getService }: FtrProviderContext) {
       after(() => esArchiver.unload('x-pack/test/functional/es_archives/infra/6.6.0/docker'));
```

Comment:
Do we really need to run a test against 6.0.0?

Reply:
same as https://github.com/elastic/kibana/pull/146573#discussion_r1048899404